Example/Example Swift/Info.plist

```diff
@@ -49,7 +49,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.9.0</string>
+	<string>0.9.1</string>
```

Comment:
I think the example should soon reach version 1.

Reply:
I think it is not a big deal since it is the example app, not the showcase app (no downloads so far)